src/Library/Library.blp

```diff
@@ -59,6 +59,18 @@ Adw.Window window {
             }
           }
 
+          Box {
+            DropDown dropdown_language {
+              halign: end;
+              margin-end: 12;
+              hexpand: true;
+            }
+
+            DropDown dropdown_category {
+              halign: end;
+            }
+          }
+
           ListBox library_uncategorized {
```

Comment:
The empty state here could probably use something:

![Screenshot of the library window, showing no results in search](https://github.com/user-attachments/assets/93f728d5-ca7d-4b8a-9a8c-93c33a72eaee)

If it were just one ListBox you'd use the placeholder feature, however the [HIG says](https://developer.gnome.org/hig/patterns/nav/search.html#search-results) *"Often a simple “No results” label is sufficient"*. Maybe one could be shown if there are no results?

Reply:
That can be done yes

Reply:
That looks good, maybe add one of the [title style classes](https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/style-classes.html#typography-styles) if one looks good, otherwise it can be tweaked it later :+1: